src/@types/map.d.ts

```diff
@@ -24,3 +14,30 @@ interface MapBaseStyles {
     navigationDay: 'mapbox://styles/mapbox/navigation-day-v1';
     navigationNight: 'mapbox://styles/mapbox/navigation-night-v1';
 }
+
+type MapLayer = LayerProps;
+type GeoJson = Feature<Geometry, GeoJsonProperties> | FeatureCollection<Geometry, GeoJsonProperties>;
+
+interface MapGeoJsonData {
```

Comment:
This is like an intermediary-data-structure that adds a bit of fluff around the raw geojson data that helps facilitate other map-like things like togglePanel elements and layer color, etc. More support-data can go in here if we decide to go this route. 

I'd like `data` to ultimately be able to take different types of data, GeoJson for one, or another could be `Indicator` (or maybe better named as 'Point') which is something like a sole X,Y coordinate representing a point

Reply:
Oh thanks for calling this out—now I understand what you meant in the meeting about always having names and such. Maybe we could base this off the spreadsheet of data sources?